src/conduits/RwaOutputConduit3.sol

```diff
@@ -325,7 +333,7 @@ contract RwaOutputConduit3 {
         require(to != address(0), "RwaOutputConduit3/to-not-picked");
 
         // We might lose some dust here because of rounding errors. I.e.: USDC has 6 dec and DAI has 18.
-        uint256 gemAmount = getGemBuyAmount(wad, psm.tout());
+        uint256 gemAmount = this.daiToGem(wad);
```

Comment:
If `daiToGem` is meant to be used by this contract, make it `public` instead of `external`.

When you call it like `this.daiToGem()` you end up spending more gas in the `CALL` than you would copying the inputs into memory.

Reply:
Done